pdns/pdnsutil.cc

```diff
@@ -2081,7 +2081,7 @@ try
       return 0;
     }
     try {
-      SSQLite3 db(cmds[1], true); // create=ok
+      SSQLite3 db(cmds[1], "WAL", true); // create=ok
```

Comment:
Why is this WAL hard-coded and not imported from pdns.conf or the command line?
Ps I'm not even sure you need a journal-mode when you create a database.

Reply:
Because any operations after this one will then also use WAL, even if not done through pdnsutil. 
If we take it from pdns.conf, most users will also get WAL.

It's not super important, if you think it's confusing or surprising, we can take it out.

Reply:
Yes, I think it is confusing. The docs say about journal-mode "Set to empty to leave the journal mode alone." Which is true for existing databases. But for new ones you silently default to WAL regardless of journal-mode being empty or not.

If you don't set the journal-mode to WAL at create time it will happen after the first connection to the db from powerdns or pdnsutil. This time the value depends on the settings in dns.conf or command line.

Reply:
Ok! Pushed!